### ISSUE TITLE
Modify query interface to support dynamic z-score calculation

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#1793]
- For expression profiles, display all profiles (except z-score profiles) no matter what's the value of "SHOW_IN_ANALYSIS_TAB"
- Add option to allow user to use `Z-scores`
- When a user choose `Z-scores`, allow to pick the reference population: `All profiled samples` or `Samples with diploid copy number of the gene in question`.

**Comment by pieterlukasse** [comment]
@jjgao I would like to support the option to have normal samples loaded for some private/internal studies. In this case, it would be nice to give the user the option to pick the reference population from: `All normal samples`. 

For the `SHOW_IN_ANALYSIS_TAB`, please remember to remove this constraint from the validator and also update the documentation in this case.

**Comment by fedde-s** [comment]
Yup, sounds like an improvement to me! Simplifies data loading and the profile table, ensures that the calculation is done in a consistent way based on the loaded data, and with the last option, potentially makes the meaning of the Z-scores more transparent to the user.

**Comment by fedde-s** [comment]
See [my comment on issue #1796](https://github.com/cBioPortal/cbioportal/issues/1796#issuecomment-253451322) for a few specific places that need to be updated as @pieterlukasse's comment [above](https://github.com/cBioPortal/cbioportal/issues/1793#issuecomment-253447392)

**Comment by yichaoS** [comment]
Done with this commit. https://github.com/yichaoS/cbioportal/commit/6c617e4496edffee99f71d42de1f84613a905d65
Can't create PR for this issue alone. Will create a PR once this is done https://github.com/cBioPortal/cbioportal/issues/1794

Below is a screenshot of the new UI
![screen shot 2016-11-09 at 7 08 17 pm](https://cloud.githubusercontent.com/assets/5973438/20159628/e7b5fed4-a6af-11e6-8445-bf602213317c.png)

**Comment by jjgao** [comment]
@schultzn any comments on @yichaoS's mock-up above?
